webpack/webpack.config.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
   output: {
     filename: `lazysizes-${gitHash}.js`,
     library: 'lazysizes',
-    libraryTarget: 'umd',
+    libraryTarget: 'var',
```

Comment:
One thing is the way we import the lazysizes library, other thing is the way we generate our library.
This variable should keep being **umd** AFAIK

Reply:
why? that's exactly the problem @thet wants to fix.

Reply:
yes, ``umd`` generates the wrapper which I wanted to get rid of. So both - including the non-umd version and changing the libraryTarget here are necessary.